docker/dockerfiles/Dockerfile.worker

```diff
@@ -1,3 +1,12 @@
+FROM golang as ecr-login-installation
```

Comment:
Changes to this file are irrelevant, I just based this branch off of my other PR branch since that'll hopefully be merged soon.